src/components/body/ammo-body.js

```diff
@@ -420,8 +424,8 @@ let AmmoBody = {
       const quaternion = this.msTransform.getRotation();
 
       const el = this.el,
-        parentEl = el.parentEl,
         body = this.body;
+      parentEl = el.object3D.parent.el ? el.object3D.parent.el : el.parentEl;
```

Comment:
Can you please add a comment above this line, something like
`// allow for THREE.js scene graph to vary from the HTML scene graph, el.object3D may have been moved to a different parent, so object3D.parent.el may not be the same as el.parentEl anymore`